.github/workflows/check-packages.yml

```diff
@@ -85,4 +85,5 @@ jobs:
             core.exportVariable('BEACHBALL_VERSION', beachballVersion);
 
       - run: |
-          npx beachball@$BEACHBALL_VERSION check
+          npx beachball@$BEACHBALL_VERSION check -b web-components-v3
+          node ./scripts/beachball/check-wc-3-changefiles
```

Comment:
should remove these and favor whats currently on `master` branch

Reply:
Would these lines still be needed while `web-components-v3` is being treated as a feature branch?

Reply:
If this is being merged to `master` then they shouldn't be needed, it's even documented in the script itself 

https://github.com/microsoft/fluentui/blob/3287714774098eb5f902d9110a214033807d5e34/scripts/beachball/check-wc-3-changefiles.js#L22-L28

Reply:
Good to know 👍 Since this particular rebase branch isn't actually going into `master` (it will be force-pushed over `web-components-v3`), I'll restore these commands in `check-packages.yml` here.